packages/terra-clinical-onset-picker/tests/nightwatch/component/LessMonth.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Base from 'terra-base';
+import moment from 'moment';
+import OnsetPicker from '../../../lib/OnsetPicker';
+
+const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
+
+export default class LessMonth extends React.Component {
+  constructor(props) {
+    super(props);
+
+    // Lock date to September 20, 2016
+    moment.now = () => new Date(2016, 8, 20);
+
+    this.state = {
+      precision: 'on/at',
+      granularity: 'MONTH',
+      onsetDate: moment(),
+    };
+
+    this.handleGranularity = this.handleGranularity.bind(this);
+    this.handlePrecision = this.handlePrecision.bind(this);
+    this.handleOnset = this.handleOnset.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleGranularity(granularity) {
+    this.setState({ granularity });
+  }
+
+  handlePrecision(precision) {
+    this.setState({ precision });
+  }
+
+  handleOnset(onsetDate) {
+    this.setState({ onsetDate });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+
+    this.setState({
+      submittedData: {
+        precision: this.state.precision,
+        granularity: this.state.granularity,
+        onsetDate: this.state.onsetDate,
+      },
+    });
+  }
+
+  render() {
+    return (
+      <Base locale={locale}>
+        <p>Controlled Onset picker with minimum required fields, supplied with a birthdate of 3 weeks old.</p>
```

Comment:
I feel like you should remove references to a "Controlled" Onset picker. You can't truly make it a controlled component since there's no way for consumers to control the state of the input values, so I think using that terminology will cause confusion.

Reply:
I've removed 'Controlled', replacing the example as a 'Handled' OnsetPicker and the test name in regards to the birthdate passed to the OnsetPicker.

Reply:
@doogthemushroom  You still have Controlled in this description.